gnucash/import-export/import-main-matcher.c

```diff
@@ -553,9 +565,18 @@ on_matcher_ok_clicked (GtkButton *button, GNCImportMainMatcher *info)
         if (first_tran)
         {
             Split* first_split = gnc_import_TransInfo_get_fsplit (trans_info);
-            xaccAccountSetAppendText (xaccSplitGetAccount(first_split), append_text);
+            Account* account = xaccSplitGetAccount (first_split);
+            xaccAccountSetAppendText (account, append_text);
             first_tran = false;
+            begin_edit (&accounts_modified, account);
```

Comment:
Tangential to your change - this test for `first_tran` assumes an import is always into a single account. This is true for typical bank imports, but is it no longer true for generic csv imports. It should do this for all source accounts we're importing into.

As said, not directly related to your PR, I just noticed it while reviewing.

Reply:
Yes, I'm not too familiar with multi-account csv imports, hence I'd sought your expertise. I agree we shouldn't limit to first split. Would it be sufficient to analyze all import transactions->splits->account?